healthcare_demo.dashboard.lookml

```diff
@@ -0,0 +1,9 @@
+- dashboard: healthcare_demo
```

Comment:
Let's remove this lkml file for now, generally you want to create a LookML dashboard if we need to version control or ensure no one can manipulate it. We can showcase this feature once we have a fully defined dashboard